plugin/config/suppressions.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+
+<!DOCTYPE suppressions PUBLIC
+  "-//Checkstyle//DTD SuppressionFilter Configuration 1.2//EN"
+  "https://checkstyle.org/dtds/suppressions_1_2.dtd">
+
+<suppressions>
+  <suppress checks="AvoidStarImport" files=".*Test\.java"/>
```

Comment:
not really sure this exclusion gives much value?

Reply:
star imports in test files I don't mind but if you disagree let's avoid start import everywhere 😅

Reply:
what do you think about applying it on all modules except snakeyaml?

Reply:
I would prefer one rule everywhere, simpler to reason with and not sure if you could configure the IDE to know in test files its fine to star import

Reply:
what's the reason for not doing it in snakeyaml?
fine with it if it would be annoying there

Reply:
You cannot I had to edit my IDE config to optimize once for tests and once for main 😰

Reply:
We have snakeyaml included so we can shade it and makes no sense to maintain the code.

I would like if possible to get RID of snakeyaml and shade it differently.
@oleg-nenashev I can't remember, but you had some suggestion on how to shade it.

Reply:
can't you do `maskClasses` on the maven-hpi-plugin?
or does that do something else?

can certainly exclude the snake yaml module then...